### ISSUE TITLE
Generated Ark comm plot_comm.rs has manual edits

### ISSUE THREAD

**Issue by wesm** [posit-dev/positron#3852]
Running the comm generator modifies plot_comm.rs (and removing the manual edit results in compilation failure):

```diff
diff --git a/crates/amalthea/src/comm/plot_comm.rs b/crates/amalthea/src/comm/plot_comm.rs
index dd5cb542..89f6a320 100644
--- a/crates/amalthea/src/comm/plot_comm.rs
+++ b/crates/amalthea/src/comm/plot_comm.rs
@@ -22,7 +22,7 @@ pub struct PlotResult {
 }
 
 /// Possible values for Format in Render
-#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, strum_macros::Display)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 pub enum RenderFormat {
   #[serde(rename = "png")]
   Png,
@@ -108,3 +108,4 @@ pub enum PlotFrontendEvent {
   Show,
 
 }
+
```

This is a small annoyance when updating the comms from the OpenRPC definitions, but it would be good to fix.

**Comment by timtmok** [comment]
This is not quite ready to test. Just need to update ark in Positron.

**Comment by timtmok** [comment]
Just merged ark 0.1.116

**Comment by wesm** [comment]
I think this is good to close
